common/v2/features/SendAssets/components/SendAssetsForm.tsx

```diff
@@ -310,6 +313,11 @@ export default function SendAssetsForm({ txConfig, onComplete }: IStepComponentP
                 value: unstoppableAddress
               });
             } catch (err) {
+              // Force the field value to error so that isValidAddress is triggered!
+              setFieldValue('address', {
+                ...values.address,
```

Comment:
This is strange. It looks like we're spreading the address which will result in `{ 0: '0', x: 'x',  ...etc }`

Reply:
Address is an object which contains `display` and `value`.